### PR TITLE
Document KCP Location API usage

### PR DESCRIPTION
**Pull request by bnallapeta**

### CODE REVIEW

**Review by ramessesii2 — APPROVED** [review]
Tested it, looks good to me.

File: gitops/README.md [hunk 1]
```diff
@@ -52,3 +52,69 @@ The PR should have:
...
+
+## Next steps
+
+Once you have completed the above process, you now have access to a kcp workspace synced with a workload cluster where you can start creating PipelineRuns.
+Furthermore, if you wish to create new workspaces (eg: dev, qe, sre etc), you can create an APIBinding on that workspace to bind to the same workload cluster that you have available. This is possible as kcp creates a APIExport Custom Resource in the workspace where the workload cluster was synced. This mechanism of using APIExport and APIBindings makes it possible to use the same compute (workload cluster) with multiple workspaces.
+
+Below snippet shows the default APIExport CR named 'kubernetes' created in a workspace named 'compute'. 
+
+```
+$ KUBECONFIG=kcpinstance.kubeconfig k ws
```
Comment (fgiloux) [inline gitops/README.md:65]: please use kubectl instead of the "k" shortcut, it may be clearer for some readers

File: gitops/README.md [hunk 2]
```diff
@@ -52,3 +52,69 @@ The PR should have:
...
+Current workspace is "root:rh-sso-${ORG_ID}:compute".
```
Comment (fgiloux) [inline gitops/README.md:66]: I would remove "rh-sso", have instead: "root:${ORG}:compute"

File: gitops/README.md [hunk 3]
```diff
@@ -52,3 +52,69 @@ The PR should have:
...
+
+#Below apiexport CR named 'kubernetes' is created automatically once we sync the workload cluster to the workspace 
+$ KUBECONFIG=kcpinstance.kubeconfig k get apiexports
+NAME         AGE
+kubernetes   46h
+ 
+$ KUBECONFIG=kcpinstance.kubeconfig k get apiexports/kubernetes -o yaml | k neat
```
Comment (fgiloux) [inline gitops/README.md:73]: I have not seen us referencing the "kubectl-neat" plugin anywhere. I would just remove its use here.

**Review by fgiloux — COMMENTED** [review]
It looks good, just a couple of possible amendments

File: gitops/README.md (hunk 3 above)
Comment (bnallapeta) [reply gitops/README.md:73]: Done.

File: gitops/README.md (hunk 2 above)
Comment (bnallapeta) [reply gitops/README.md:66]: Makes sense.

File: gitops/README.md (hunk 1 above)
Comment (bnallapeta) [reply gitops/README.md:65]: Done.

File: gitops/README.md [hunk 4]
```diff
@@ -52,3 +52,69 @@ The PR should have:
 
 - kubeconfig/s of the OpenShift clusters to be placed under gitops/credentials/kubeconfig/compute
 - overlay/kustomization.yaml to be created under `gitops/environment/compute/<clustername>/overlay/kustomization.yaml`
+
+
+## Next steps
+
+Once you have completed the above process, you now have access to a kcp workspace synced with a workload cluster where you can start creating PipelineRuns.
+Furthermore, if you wish to create new workspaces (eg: dev, qe, sre etc), you can create an APIBinding on that workspace to bind to the same workload cluster that you have available. This is possible as kcp creates a APIExport Custom Resource in the workspace where the workload cluster was synced. This mechanism of using APIExport and APIBindings makes it possible to use the same compute (workload cluster) with multiple workspaces.
```
[the remark this answers is not in the document]
Comment (bnallapeta) [reply gitops/README.md:60]: Done.

**Pull request merged**